src/backup/backup_common.h

```diff
@@ -28,4 +54,18 @@ void backup_cleanup_backup_data(BackupData* backup_data);
  */
 void backup_calculate_checksum(BackupContent* content, BackupData* backup_data, uint8_t* hash);
 
+/**
+ * Fills the backup structure with backup data.
+ * @param[in] backup_create_timestamp The time at which the backup was created.
+ * @param[out] backup The backup structure filled with data.
+ * @param[out] backup_data The backup data structure filled with data.
+ * @param[out] encode_data Additional data required for encoding/decoding.
+ */
+backup_error_t backup_fill(
+    uint32_t backup_create_timestamp,
+    uint32_t seed_birthdate_timestamp,
```

Comment:
Missing param doc for birthdate. I know you just moved it around but since you're refactoring, probably a good time to update.

Reply:
fixed